pkg/collector/helm2_test.go

```diff
@@ -0,0 +1,28 @@
+package collector
+
+import (
+	"testing"
+
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func TestNewHelmV2Collector(t *testing.T) {
+	expectedName := "Helm v2"
```

Comment:
I'd consider making this a const in pkg/collector/helm2.go, to make the tests less brittle

Reply:
done